tsdb/index/index.go

```diff
@@ -523,6 +523,11 @@ func (w *Writer) AddSymbol(sym string) error {
 }
 
 func (w *Writer) finishSymbols() error {
+	// Right now, the index symbol_table's <len> part is 4byte length. So the total symbol string length must less than 4byte
```

Comment:
We follow this comment style, with some suggestions
```suggestion
	// The symbol table's <len> part is 4 bytes. So the total symbol table size must be less than or equal to 2^32-1.
```